chef/cookbooks/updater/recipes/default.rb

```diff
@@ -40,6 +40,31 @@
       zypper_command += " --auto-agree-with-licenses"
     end
 
+    ruby_block "check for HA updates" do
+      block do
+        case node[:updater][:zypper][:method]
+        when "patch"
+          command = "zypper list-patches"
+        when "update"
+          command = "zypper list-updates"
+        when "dist-upgrades"
+          # TODO: check the correct one for here
+          command = "zypper list-updates"
+        end
+        command += '|egrep "corosync|pacemaker" -q'
+        `#{command}`
+        node.run_state["found_ha_packages"] = $?.exitstatus
+      end
+      not_if { search(:node, "roles:pacemaker-cluster-member").empty? }
+    end
+
+    ["corosync", "pacemaker"].each do |s|
```

Comment:
Even though it seems necessary to do this at the moment, it shouldn't be.  I've filed [bsc#991989 (pacemaker.service should have BindsTo=corosync.service)](https://bugzilla.suse.com/show_bug.cgi?id=991989) for this.